src/index.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
```

Comment:
You don't use this class anywhere. Therefore, it's useless.
```suggestion
<html lang="en">
```